tests/Tests/600__ai_apps/680__starburst/681__managed_starburst.robot

```diff
@@ -1,9 +1,36 @@
 *** Settings ***
 Documentation    Suite to test Managed Starburst integration
-Resource    ../../../Resources/Page/ODH/AiApps/ManagedStarburst.resource
+Resource         ../../../Resources/Page/ODH/AiApps/ManagedStarburst.resource
+Resource         ../../../Resources/Page/LoginPage.robot
+Resource         ../../../../tasks/Resources/SERH_OLM/install.resource
+Suite Setup      Starburst Setup Suite
+Suite Teardown    RHOSi Teardown
+
+
+*** Variables ***
+${GET_SQL_FUNC}=           import pandas\ndef get_sql(sql, connector):\n\tcur = connector.cursor()\n\tcur.execute(sql)\n\treturn pandas.DataFrame(cur.fetchall(), columns=[c[0] for c in cur.description])\nprint("get_sql function defined")    # robocop: disable
+${INIT_CELL_CODE}=         import os\nimport trino\nTRINO_USERNAME="dummy-user"\nTRINO_HOSTNAME = os.environ.get('TRINO_HOSTNAME')\nTRINO_PORT= 80\nconn = trino.dbapi.connect(\nhost=TRINO_HOSTNAME,\nport=TRINO_PORT,\nuser=TRINO_USERNAME\n)\nprint("connection to trino set")    # robocop: disable
+${QUERY_CATALOGS}=         SHOW CATALOGS
+${QUERY_SCHEMAS}=          SHOW SCHEMAS from tpch
+${QUERY_TABLES}=           SHOW TABLES from tpch.sf1
+${QUERY_CUSTOMERS}=        SELECT name FROM tpch.sf1.customer limit 3
+${QUERY_JOIN}=             SELECT c.name FROM tpch.sf1.customer c JOIN tpch.sf1.orders o ON c.custkey = o.custkey ORDER BY o.orderdate DESC limit 3    # robocop: disable
+${QUERY_CATALOGS_PY}=      sql = '${QUERY_CATALOGS}'\ndf = get_sql(sql, conn)\nprint(df['Catalog'].values)\n
+${QUERY_SCHEMAS_PY}=       sql = '${QUERY_SCHEMAS}'\ndf = get_sql(sql, conn)\nprint(df['Schema'].values)\n
+${QUERY_TABLES_PY}=        sql = '${QUERY_TABLES}'\ndf = get_sql(sql, conn)\nprint(df['Table'].values)\n
+${QUERY_CUSTOMERS_PY}=     sql = '${QUERY_CUSTOMERS}'\ndf = get_sql(sql, conn)\nprint(df['name'].values)\n    # robocop: disable
+${QUERY_JOIN_PY}=     sql = '${QUERY_JOIN}'\ndf = get_sql(sql, conn)\nprint(df['name'].values)\n    # robocop: disable
 
 
 *** Test Cases ***
+Verify Managed Starburst Is Deployed
+    [Documentation]    Checks that Managed Starburst is deployed in the cluster
+    [Tags]    MISV-79    MISV-84
+    ${installed}=    Is Managed Starburst Installed
+    Should Be Equal    ${installed}    ${TRUE}
+    Status Of Managed Starburst CR Should Be    Deployed
+    Managed Starburst Pods Should Be Deployed
+
```

Comment:
Should you have a keyword that skips the next TCs if Starburst is not installed or deployed correctly?

Reply:
we could yeah, but at the end the result would be the same, meaning that the next TCs would fail at the very first kw, like I think happens with GPU if GPU Addon is not installed.

I can skip however, but I would do in another PR later if not a problem

Reply:
yep sure, not urgent.
Agreed that it's not a big deal since it would fail soon, but for autoscalers we implemented the skip keyword and I think it's cleaner this way rather than having a preventable failure